### ISSUE TITLE
Prototype 1.3.0 does not work on Mac Os 10.13

### ISSUE THREAD

**Issue by Xenakios** [VCVRack/VCV-Prototype#42]
Rack's log shows :

```
[0.109 warn src/plugin.cpp:158] Could not load plugin /Users/Me/Documents/Rack/plugins-v1/VCV-Prototype: Failed to load library /Users/Me/Documents/Rack/plugins-v1/VCV-Prototype/plugin.dylib: dlopen(/Users/Me/Documents/Rack/plugins-v1/VCV-Prototype/plugin.dylib, 6): Symbol not found: ____chkstk_darwin
  Referenced from: /Users/Me/Documents/Rack/plugins-v1/VCV-Prototype/plugin.dylib
  Expected in: /usr/lib/libSystem.B.dylib
 in /Users/me/Documents/Rack/plugins-v1/VCV-Prototype/plugin.dylib
```

**Comment by AndrewBelt** [comment]
Does this build load on the same machine?
[VCV-Prototype-1.3.0-mac.zip](https://github.com/VCVRack/VCV-Prototype/files/4967432/VCV-Prototype-1.3.0-mac.zip)
Let me know ASAP so I can get this fix out immediately.

**Comment by Xenakios** [comment]
> Does this build load on the same machine?
> [VCV-Prototype-1.3.0-mac.zip](https://github.com/VCVRack/VCV-Prototype/files/4967432/VCV-Prototype-1.3.0-mac.zip)
> Let me know ASAP so I can get this fix out immediately.

Yes, that build loads and runs.

**Comment by AndrewBelt** [comment]
Build fixed and released as a hotfix with the same version 1.3.0.
